modules/ding_place2book/ding_place2book.install

```diff
@@ -178,10 +178,79 @@ function ding_place2book_update_7005(&$sandbox) {
   variable_del('ding_place2book_category_mappings');
 }
 
-
 /**
  * Enable migration module for existing sites.
  */
 function ding_place2book_update_7006() {
   module_enable(array('ding_place2book_migrate'));
 }
+
+/**
+ * Remove ticket links from existing passive events.
+ */
+function ding_place2book_update_7007(&$sandbox) {
+  $nodes = [];
+
+  // Get the ding_event nodes we need to consider. We only consider those with
+  // an event end-date that hasn't passed. We can't make any assumptions about
+  // whether the nodes are publsihed or not, so no condition on $node->status.
+  $query = new EntityFieldQuery();
+  $query->entityCondition('entity_type', 'node')
+    ->entityCondition('bundle', 'ding_event')
+    ->fieldCondition('field_ding_event_date', 'value2', gmdate('Y-m-d H:i:s'), '>');
```

Comment:
Instead of this string we should probably use `DATE_FORMAT_DATETIME`.

Also: Is there any reason why you use `gmdate()` I has expected `date` to be sufficient. `gmdate()` does not seem to be used anywhere within the date module itself.

Reply:
The date string are in UTC in the database. gmdate ensures that we get the current time as datetime string in UTC.

Reply:
Will use DATE_FORMAT_DATETIME

Reply:
Fixed!